[IMP] Misc: 15.0 email templates / FAQ update

tiku-odoo opened the pull request:
@odoo/us-doc-review 

When you have a moment, can you review this doc?

**This doc is only for V15. V16 has additional changes.** 

**I Spoke with Sam, and he advised me to forward port over to Odoo 16 and make the additional edits afterward. (added 16.0 tag). Additional information has to be inserted for dynamic placeholders. Everything else holds true.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/4229).

StraubCreative commented:
@tiku-odoo CI checks failing, [details here](https://runbot.odoo.com/runbot/build/30524741).

Can you also squash into 1 commit please?

Thanks lmk if you need help with anything 👍

tiku-odoo commented:
@StraubCreative  I've fixed the build errors and squashed the two commits. Thanks for catching that! :1st_place_medal:

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+**Basic Blocks**
+
+- Three different heading types
+- A text paragraph block
+- A bulleted list
+- A numbered list
+- A checklist
+- A table
+- The ability to switch the direction of the text
+- A horizontal separator
+- A quote
```
I would remove the "A" from the start of any bullet that includes it

tiku-odoo replied:
Thanks, this is a good point.

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+
+- Image
+
+To activate one of these options simply click on the option in the Powerbox. If it is a text option
+then highlight the text first and type in the activator key `/` and select the option.
```
```suggestion
To activate one of these options, simply click on it in the Powerbox. If it is a text option,
then highlight the text, type in the activator key `/`, and select the option.
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+Dynamic placeholders are encoded to display fields from within the database. Dynamic placeholders
+can no longer be used in Odoo 15 except for in the subject line, and to and from fields. Although
+the generator is present as the last tab on the template, it will create placeholders for use only
+in the above mentioned fields. Dynamic placeholders may be inserted in the HTML code, but this task
```
```suggestion
in the above-mentioned fields. Dynamic placeholders may be inserted in the HTML code, but this task
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+.. seealso::
+   :doc:`../../../../services/support/what_can_i_expect`
+
+Defining a default reply to on the mail template
+================================================
```
```suggestion
Defining a default reply on the mail template
=============================================
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+Defining a default reply to on the mail template
+================================================
+
+Although the *reply to* field is available within the mail templates, **this field is only used
+for mass mailing** mode (this means when sending templates on bulk emailing). Emails
```
```suggestion
Although the *reply to* field is available within email templates, **this field is only used
for mass mailing** mode (when sending emails in bulk). Emails
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+Although the *reply to* field is available within the mail templates, **this field is only used
+for mass mailing** mode (this means when sending templates on bulk emailing). Emails
+can be sent in bulk in almost every application that has a list view. Select the records where the
+emails are to be sent and click on the :guilabel:`Action` button. If there is the option to send an
+email, a mail composer with possible values to define will appear.
```
```suggestion
emails are to be sent and click on the :guilabel:`Action` button. If it is possible to send an
email, a mail composer with values that can be defined will appear.
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
    :alt: Composer in mass mailing mode after selecting multiple quotations.
 
-You can also define them by default on the template:
+They can also be defined by default on the template:
```
```suggestion
Values can also be defined by default on the template:
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
    :alt: Reply-to field on template.
 
 Because of this, setting a value in this field is useless as the value defined will be totally
 ignored. The default *reply-to* value is the default catchall email address to ensure a
```
```suggestion
ignored. The default *reply-to* value is the default catchall email address, to ensure a
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+the catchall works, check :ref:`how to manage inbound messages
+<email_communication/inbound_messages>`.
 
 Transactional emails and corresponding URL for each company
 ===========================================================
```
```suggestion
Transactional emails and corresponding URLs for each company
============================================================
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
 Transactional emails and corresponding URL for each company
 ===========================================================
 
 When using Odoo, multiple events trigger the sending of automated emails. These emails are known
```
```suggestion
In Odoo, multiple events can trigger the sending of automated emails. These emails are known
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+as transactional emails and sometimes contain links pointing to the Odoo database.
 
 By default, links generated by the database use the dynamic web.base.url key defined in the system
 parameters. More information about this :ref:`parameter <domain-name/web-base-url>`.
```
```suggestion
parameters. For more information about this, see :ref:`parameter <domain-name/web-base-url>`.
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
-It’s important to know that this key can only have a single value, meaning that in a
-multi-website/company database environment, even if you have a specific domain name for each
-website, the links generated to share a document or within a transactional email might remain the
+It's important to know that this key can only have a single value, meaning that in a
+multi-website/company database environment, even there is a specific domain name for each
+website, the links generated to share a document or within a transactional email may remain the
```
I don't really understand what this is saying. I would rewrite it entirely.

tiku-odoo replied:
I've added an example: 

.. example::
   If the value of the web.base.url system parameter is equal to ``https://www.purpletest.com`` and
   there are two separate websites in Odoo with different URLS: ``https://www.purplecompany.com``
   and ``https://www.purpleteam.com``, the links created by Odoo to share a document or send a
   transactional email will come from the domain: ``https://www.purpletest.com``.

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +136,35 @@ website of the company.
    limitation!
 
-On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a
+On the other hand, sales orders made by a customer on one of the Odoo e-commerce websites have a
```
```suggestion
On the other hand, sales orders made by a customer on an Odoo e-commerce website have a
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +136,35 @@ website of the company.
-On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a
+On the other hand, sales orders made by a customer on one of the Odoo e-commerce websites have a
 link established with the website from which the order was made. As a result, the e-mail sent for
 the sales orders uses the domain name defined for the corresponding website to generate the links.
```
```suggestion
sales orders uses the domain name defined for the corresponding website to generate the links.
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +136,35 @@ website of the company.
-Email templates are automatically translated. Changing the translations shouldn’t be necessary.
-However, if for a specific reason you’d like to change some of the translations, this can be done.
+Email templates are automatically translated. Changing the translations shouldn't be necessary.
+However, if for a specific reason that some of the translations need to be changed, it can be done.
```
```suggestion
However, if for a specific reason, some of the translations need to be changed, it can be done.
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +136,35 @@ website of the company.
 
-Like any modification in the code, keep in mind that modifications that aren’t done correctly (for
-example modifications leading to bad syntax) can break the template, as a result, the template
+Like any modification in the code, keep in mind that modifications that aren't done correctly (for
+example modifications leading to bad syntax) can break the template, and as a result, the template
 will appear blank.
```
```suggestion
Like any modification in the code, keep in mind that if they aren't done correctly (for
example, modifications leading to bad syntax), it can break the template, and as a result, the template
will appear blank.
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +136,35 @@ website of the company.
 will appear blank.
 
-In order to edit your translations, follow these steps from the template.
+In order to edit the translations, follow these steps from the template:
```
```suggestion
In order to edit translations, follow these steps from the template:
```

hojo-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +136,35 @@ website of the company.
 #. A pop-up window with the different languages installed on the database will be displayed. From
-   here, editing the translations will be possible. Don't forget to hit the save button to preserve
-   your changes.
+   here, editing the translations will be possible. Don't forget to hit the save button to save
```
```suggestion
   here, editing the translations will be possible. Don't forget to hit the :guilabel:`Save` button to save
```

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+and text style or add XML/HTML features in an email template. The Powerbox is activated by typing a
+forward slash `/` in the body of the email template.
+
+The following options will be available once the Powerbox is activated:
```
This is noted in the line before. Line 27/28. All that needs to be done is typing a forward slash.

I've changed the wording to: The following options will be available once a forward slash `/` is typed in:

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,56 +1,128 @@
+
+**Basic Blocks**
+
+- Three different heading types
```
I've added these in: (H1-H3: Small-Large)

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +141,35 @@ website of the company.
-On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a
-link established with the website from which the order was made. As a result, the e-mail sent for
-the sales orders uses the domain name defined for the corresponding website to generate the links.
+On the other hand, sales orders made by a customer on an Odoo e-commerce website have a link
+established with the website from which the order was made. As a result, the e-mail sent for sales
+orders uses the domain name defined for the corresponding website to generate the links.
```
```suggestion
On the other hand, sales orders made by a customer, on an Odoo eCommerce website, have a link established with the website, from which the order was made. As a result, the e-mail sent for sales orders uses the domain name defined for the corresponding website to generate the links.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +141,35 @@ website of the company.
 
-Email templates are automatically translated. Changing the translations shouldn’t be necessary.
-However, if for a specific reason you’d like to change some of the translations, this can be done.
+Email templates are automatically translated. Changing the translations shouldn't be necessary.
+However, if for a specific reason, some of the translations need to be changed, it can be done.
```
```suggestion
In Odoo, email templates are automatically translated. Changing the translations shouldn't be necessary.
However, if for a specific reason, some of the translations need to be changed, it can be done.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +141,35 @@ website of the company.
-Like any modification in the code, keep in mind that modifications that aren’t done correctly (for
-example modifications leading to bad syntax) can break the template, as a result, the template
-will appear blank.
+Like any modification in the code, keep in mind that if they aren't done correctly (for example,
+modifications leading to bad syntax), it can break the template, and as a result, the template will
+appear blank.
```
```suggestion
However, like any modification in the code, if they aren't done correctly (for example, modifications leading to bad syntax), it can break the template, and as a result, the template will appear blank.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +141,35 @@ website of the company.
+In order to edit translations, follow these steps from the template:
 
-#. Click on the edit button, then on the language button
+#. Click on the :guilabel:`edit` button, then on the language button
```
```suggestion
#. Click on the :guilabel:`Edit` button, then on the language button, represented by the initials of the language currently being used.
```
added some specificity

tiku-odoo replied:
Thanks KC!

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +141,35 @@ website of the company.
+      :align: center
+      :alt: Edit the language of a template.
 
 #. A pop-up window with the different languages installed on the database will be displayed. From
```
```suggestion
#. A pop-up window with the different languages installed on the database appears. From
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -64,35 +141,35 @@ website of the company.
 #. A pop-up window with the different languages installed on the database will be displayed. From
-   here, editing the translations will be possible. Don't forget to hit the save button to preserve
-   your changes.
+   here, editing the translations will be possible. Don't forget to hit the :guilabel:`Save` button
+   to save the changes.
```
```suggestion
   this pop-up, the editing of translations is possible. When the desired changes have been made, click the 
   :guilabel:`Save` button to save the changes.
```

ksc-odoo reviewed (commented):
Hey @tiku-odoo - just finished peer reviewing this doc. Great work. If you have any questions at all, or need clarification about any of the suggested edits, don't hesitate to reach out. Thanks!

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
-can send emails in bulk in almost every app that has a list view. Select the records you want
-and click on the action button. If you have an option to send an email, you will see a mail
-composer with possible values to define:
+Starting with Odoo 15, the Powerbox feature can be used. The Powerbox allows for the user to edit
+the format of the text, to add links or buttons, appointment options, and to add images to the email
+template. Additionally, the XML/HTML code of the email template can be edited via the
+:guilabel:`</>` icon. Dynamic placeholders (referencing fields within Odoo) are only available in
+the subject line and to and from fields in Odoo 15.
```
I confirmed this and added in the proper fields. Thanks for pointing this out.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
+
+**Basic Blocks**
+
+- Three different heading types (H1-H3: Small-Large)
+- Text paragraph block
+- Bulleted list
+- Numbered list
+- Checklist
+- Table
+- The ability to switch the direction of the text
+- Horizontal separator
+- Quote
+- Add code to the text
+- Add a specific appointment
+- Schedule an appointment
```
Nice! Thanks for adding this in!

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
+- Link
+- Button
+
+**Media**
```
Yeah, I noticed this as well. I added the fictional word in :)

Medias is actually a word to describe the plural "media" that cover news events-- The medias of the country covered the event...

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
+
+- Image
+
+To activate one of these options, simply click on it in the Powerbox. If it is a text option, then\
+highlight the text, type in the activator key `/`, and select the option.
```
thanks, great suggestion @ksc-odoo

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
+Editing via XML/HTML code
+-------------------------
+
+To access the XML/HTML editor for the email template click on the :guilabel:`</>` icon in the upper
+right hand corner of the template. The icon will appear in turquoise.
```
Great description, thanks for adding this in.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
+Dynamic placeholders
+--------------------
+
+Dynamic placeholders are encoded to display fields from within the database. Dynamic placeholders
+can no longer be used in Odoo 15 except for in the subject line, and to and from fields. Although
+the generator is present as the last tab on the template, it will create placeholders for use only
+in the above-mentioned fields. Dynamic placeholders may be inserted in the HTML code, but this task
+is out of the scope of Odoo Support.
```
Great suggestions, thanks.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
+Defining a default reply on the mail template
+=============================================
+
+Although the *reply to* field is available within email templates, **this field is only used for
+mass mailing** mode (when sending emails in bulk). Emails can be sent in bulk in almost every
+application that has a list view. Select the records where the emails are to be sent and click on
+the :guilabel:`Action` button. If it is possible to send an email, a mail composer with values that
+can be defined will appear.
```
Thanks for expanding on this.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,58 +1,135 @@
    :alt: Composer in mass mailing mode after selecting multiple quotations.
 
-You can also define them by default on the template:
+Values can also be defined by default on the template:
```
Noted, I agree with you here, I've removed.

tiku-odoo commented:
@ksc-odoo Thanks so much for reviewing the email_template.rst doc. I've made all the requested changes. When you have a moment can you review the second doc in this PR (faq.rst)

I'm sorry to lump multiple docs into one PR. I will keep this in mind for the next round of changes. 

Thanks, Tim

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
 Here are the default limits for new databases:
 
 - 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription,
 
 - 50 emails/day for one-app free and trial databases,
```
```suggestion
- 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription.

- 50 emails/day for one-app free and trial databases.
```
replaced commas with periods - unsure if the commas were necessary in this list-like format.

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
+- In case of migration, the database daily limit might be reset to 50 emails a day.
 
-In case you hit the limit, you can:
+In case the limit is reached, the following actions can be taken:
```
```suggestion
If the limit is reached, the following actions can be taken:
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
-- Ask our support team to increase your daily limit. We will analyze the situation of your database
-  depending on (non-exhaustive list):
+- Ask Odoo's support team to increase the daily limit. The Support Team will analyze the situation
+  of the database depending on (non-exhaustive list):
```
```suggestion
- Ask Odoo's support team to increase the daily limit. The Support Team analyzes the situation
  of the database depending on the following (non-exhaustive list):
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
-You can mitigate this by receiving your :ref:`notifications in Odoo <discuss_app/notification_preferences>`
-instead of by emails.
+This can be mitigated by receiving :ref:`notifications in Odoo
+<discuss_app/notification_preferences>` instead of by emails.
```
```suggestion
This can be mitigated by receiving :ref:`notifications in Odoo
<discuss_app/notification_preferences>`, instead of emails.
```
cleaned up a bit, but please make sure that the meaning is still accurate with my suggested removal of the word "by"

tiku-odoo replied:
I've changed it to "by email". Thanks for cleaning up.

tiku-odoo replied:
Sounds better, thanks.

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
-Transport Protocol (SMTP) error messages. SMTP is a protocol to describe the email structure
-and transmit it over the Internet, and the error messages generated by email services are helpful
-tools to diagnose and troubleshoot email problems.
+It's easy to find out why an email wasn't transmitted successfully by reviewing the Simple Mail
+Transport Protocol (SMTP) error messages. :abbr:`SMTP (Simple Mail Transport Protocol)` is a
+protocol to describe the email structure and transmit it over the Internet. The error messages
+generated by email services are helpful tools to diagnose and troubleshoot email problems.
```
```suggestion
It's easy to find out why an email wasn't transmitted successfully, by reviewing the Simple Mail
Transport Protocol (SMTP) error messages. 

:abbr:`SMTP (Simple Mail Transport Protocol)` is a protocol to describe the email structure and
transmits it over the Internet. The error messages generated by email services are helpful tools to
diagnose and troubleshoot email problems.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
-If you have this problem on a recurring basis with the same client or the same domain, please
-do not hesitate to contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
+If this a problem on a recurring basis with the same client or the same domain, do not hesitate to
+contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
```
```suggestion
If this a recurring problem with the same client, or the same domain, do not hesitate to
contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
-we use a delayed job to send emails that we consider as not urgent (Newsletters concept
+It may happen that an email campaign is scheduled but it is not sent on time. We know that
+Odoo uses a delayed job to send emails that are considered as not urgent (Newsletters concept
 such as mass mailing, marketing automation, events). The system utility **cron** can be used
```
```suggestion
such as: mass mailing, marketing automation, events). The system utility **cron** can be used
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
-The emails considered urgent (communication from one person to another one such as
-Sales Orders, Invoices, Purchase Orders, etc.) are sent directly.
+The emails considered urgent (communication from one person to another one such as Sales Orders,
+Invoices, Purchase Orders, etc.) are sent directly (immediately).
```
```suggestion
Emails that are considered urgent (communication from one person to another, such as: Sales Orders,
Invoices, Purchase Orders, etc.) are sent directly (immediately).
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
-By default, the Mass Mailing cron runs every 60 minutes. So, you should wait maximum an hour
-before the campaign is actually sent.
+By default, the Mass Mailing cron runs every 60 minutes. This can be changed to no less than 5
+minutes, but for the reasons mentioned above this isn't recommended. To access this setting go to:
+:menuselection:`Settings --> Technical --> Automation --> Scheduled Actions`. The action name is
+`Mail: Email Queue Manager`
```
```suggestion
By default, the Mass Mailing cron runs every 60 minutes. This can be changed to no less than 5
minutes. But, for the reasons mentioned above, this isn't recommended. To access this setting go to:
:menuselection:`Settings --> Technical --> Automation --> Scheduled Actions`. 

Select the action name, `Mail: Email Queue Manager,` and proceed to make any necessary adjustments.
```
quick question - is the Mass Mailing cron worthy of capitalization? If so, no worries. But would an Italics-route be useful here? Not quite sure - totally your call. What do you think?

tiku-odoo replied:
I think it would work with italics. Thanks for pointing this out!

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
-When you have an issue with incoming emails, there might not be an indication per se in Odoo.
-This is the client who tries to contact a database who will get a bounce (most of the
-time 550: mailbox unavailable).
+When there is an issue with incoming emails, there might not be an indication per se in Odoo. It is
+the sending email client who tries to contact a database that will get a bounce-back (most of the
+time a `550: mailbox unavailable`).
```
```suggestion
When there is an issue with incoming emails, there might not be an indication, per se, in Odoo. It
is the sending email client, who tries to contact a database, that will get a bounce-back (most of
the time a `550: mailbox unavailable`).
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 -----------------------
 
-Depending on the platform you are using:
+Depending on the Odoo platform the database is using:
```
```suggestion
The following steps are all dependent on the Odoo platform the database is using:
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
+.. tip::
+  The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh database
+  contains a list of files containing the logs of the database. The log files are created everyday
+  at 5:00 AM UTC. The two last days are not compressed, while the older ones are, in order to gain
+  space. The naming of the files for Today and Yesterday are respectively: :file:`odoo.log` and
+  :file:`odoo.log.1`. For the following days, they are named with their dates and compressed. See
+  the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and ``zgrep``
+  (for the compressed ones) to search through the files.
```
```suggestion
  at 5:00 AM (UTC).
  
  The two last days are not compressed, while the older ones are, in order to gain
  space. The naming of the files for Today and Yesterday are respectively: :file:`odoo.log` and
  :file:`odoo.log.1`.
  
  For the following days, they are named with their dates, and then compressed. See
  the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and ``zgrep``
  (for the compressed ones) to search through the files.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 ---------------------
 
 In order to get helped efficiently, please provide as much information as possible. Here is a list
 of what can be helpful:
```
```suggestion
of what can be helpful when reaching out to support about an issue:
```

ksc-odoo replied:
```suggestion
of what can be helpful when reaching out to support about an issue:
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
-  might help you on how to get your EML files. Once you get the EML of the email, adding it
-  in the attachment of your ticket is the most efficient way for us to investigate. The support
-  will mainly focus on redundant issues.
+- The **EML** or **headers** of the email, is the file format containing all the technical
+  information required for an investigation. The documentation from the email provider might provide
+  the process on how to get EML/header files. Once the headers of the email are obtained, adding it
+  into the Odoo support ticket is the most efficient way for Odoo support to investigate.
```
```suggestion
- The **EML** (or **headers**) of the email, is the file format containing all the technical
  information required for an investigation. The documentation from the email provider might provide
  the process on how to get EML/header files. Once the headers of the email are obtained, adding it
  into the Odoo support ticket is the most efficient way for the Odoo Support team to investigate.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
+  - Is this simply a reply from an email going out from Odoo?
 
-  - Are you using an incoming email server or somehow redirecting?
+  - Is there an incoming email server being used or is email somehow being redirected?
```
```suggestion
  - Is there an incoming email server being used, or is email somehow being redirected?
```

ksc-odoo reviewed (commented):
@tiku-odoo Alrighty - just finished my peer review of the Faq.rst - my apologies for not seeing it before. If you have any questions, or need clarification on anything, don't hesitate to reach out. As always, great work!

tiku-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,185 @@
 
 Outgoing emails
 ===============
 
```
Added in thanks!

tiku-odoo commented:
@hojo-odoo @ksc-odoo @odoo/us-doc-review 

The FAQ.rst and emailtemplate.rst are both ready for a final sweep from your team. 

Thanks in advance for looking these two docs over.

tiku-odoo commented:
> 

@odoo/us-doc-review 

Can you take a last look at the two documents: 
FAQ.rst
Templates.rst

Thanks.

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+the same structure repeatedly, which saves time and improves efficiency.
+
+Creating different templates that are tailored to specific situations and audiences lets users
+choose the right message for the right audience, which increases the quality of the message and
+enhances the overall engagement rate.
```
```suggestion
choose the right message for the right audience, which increases the quality of the message, and
enhances the overall engagement rate.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
-   The email templates use QWeb. The composer allows you to edit emails in their final rendering,
-   making customizations more robust as you don’t have to edit code.
+   Email templates in Odoo use QWeb or XML, which allows the user to edit emails in their final
+   rendering, making customizations more robust without having to edit any code whatsoever.
```
```suggestion
   rendering, making customizations more robust, without having to edit any code whatsoever.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+
+The *Powerbox* feature is an enriched text editor with various formatting, layout, and text options.
+It can also be used to add XML/HTML features in an email template. The powerbox feature is
+activated, by typing a forward slash `/` in the body of the email template.
```
```suggestion
activated by typing a forward slash `/` in the body of the email template.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
-   :alt: Composer in mass mailing mode after selecting multiple quotations.
+   :alt: Powerbox feature in the email template.
+
+Editing via XML/HTML code
+-------------------------
```
```suggestion
XML/HTML code editor
--------------------
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+Settings`, scroll down to :guilabel:`Developer Tools` section, and click :guilabel:`Activate the
+developer mode`.
+
+When clicked, the page will auto-refresh, return to the main Odoo dashboard, and a "bug" icon is
+present on the top of the screen, beside the login and communication icons.
```
```suggestion
When clicked, the page refreshes, returns to the main Odoo dashboard, and a "bug" icon is present on
the top of the screen, beside the login and communication icons.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+- :guilabel:`Language` (Settings Tab)
+
+While the :guilabel:`Dynamic Placeholder Generator` is present as the last tab on the template, it
+only creates placeholders for aforementioned fields. Dynamic placeholders may also be inserted in
+the HTML code, but this task is out of the scope of Odoo Support.
```
```suggestion
only creates placeholders for the aforementioned fields. Dynamic placeholders may also be inserted
in the HTML code, but this task is out of the scope of Odoo Support.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+
+To use the :guilabel:`Dynamic Placeholder Generator` navigate to the :guilabel:`Dynamic Placeholder
+Generator` tab. Select the :guilabel:`Field` that the *dynamic placeholder* should reference by
+clicking on the :guilabel:`Field` dropdown. Should a :guilabel:`Sub-Model` or :guilabel:`Sub-Field`
```
```suggestion
clicking on the :guilabel:`Field` drop-down. Should a :guilabel:`Sub-Model` or :guilabel:`Sub-Field`
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+To use the :guilabel:`Dynamic Placeholder Generator` navigate to the :guilabel:`Dynamic Placeholder
+Generator` tab. Select the :guilabel:`Field` that the *dynamic placeholder* should reference by
+clicking on the :guilabel:`Field` dropdown. Should a :guilabel:`Sub-Model` or :guilabel:`Sub-Field`
+need to be specified, set those by selecting the corresponding dropdown. The *dynamic placeholder*
```
```suggestion
need to be specified, set those by selecting the corresponding drop-down. The *dynamic placeholder*
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+clicking on the :guilabel:`Field` dropdown. Should a :guilabel:`Sub-Model` or :guilabel:`Sub-Field`
+need to be specified, set those by selecting the corresponding dropdown. The *dynamic placeholder*
+will appear in the :guilabel:`Placeholder Expression` field. Copy and paste the *dynamic
+placeholder* in the appropriate field as listed above.
```
```suggestion
placeholder* in the appropriate field, as listed above.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+will appear in the :guilabel:`Placeholder Expression` field. Copy and paste the *dynamic
+placeholder* in the appropriate field as listed above.
+
+Configuring a default reply on email templates
+==============================================
```
```suggestion
Default reply on email templates
================================
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+==============================================
+
+Under the :guilabel:`Email Configuration` tab on an email template, there is a :guilabel:`Reply To`
+field. In this field, email addresses can be added, to which replies will be redirected when sending
+emails in mass using that template.
```
```suggestion
field. In this field, email addresses can be added, to which replies are redirected when sending
emails in mass using that template.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
-communication between your customer and your Odoo database. For more information about the way
-the catchall works, please check :ref:`how to manage inbound messages <email_communication/inbound_messages>`.
+The :guilabel:`Reply To` field is **only** used for mass mailing endeavors (sending emails in bulk).
+Emails can be sent in bulk in almost every Odoo application that has a `List` view option.
```
```suggestion
Bulk emails can be sent in almost every Odoo application that has a `List` view option.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+To send mass mails (while in `List` view), select the desired records where the emails are to be
+sent, click the :guilabel:`Action` button, represented by a `Gear` icon (:guilabel:` ⚙️`), and
+select the desired email option from the :guilabel:`Action` drop-down menu. If it is possible to
+send an email, a mail composer pop-up, with values that can be defined and customized, appears.
```
```suggestion
send an email, a mail composer pop-up appears, with values that can be defined and customized.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+.. example::
+   If the value of the web.base.url system parameter is equal to `https://www.mycompany.com` and
+   there are two separate websites in Odoo with different URLS: `https://www.mycompany2.com` and
+   `https://www.mycompany1.com`, the links created by Odoo to share a document or send a
+   transactional email will come from the domain: `https://www.mycompany.com`.
```
```suggestion
   `https://www.mycompany1.com`, the links created by Odoo to share a document, or send a
   transactional email, come from the domain: `https://www.mycompany.com`.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+   `https://www.mycompany1.com`, the links created by Odoo to share a document or send a
+   transactional email will come from the domain: `https://www.mycompany.com`.
 
 This is not always the case as some Odoo applications have a link established in the database with
```
```suggestion
This is not always the case, as some Odoo applications have a link established in the database with
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
-websites, the URL generated in the email template will use the domain defined on the corresponding
-website of the company.
+the website application. In that case, if a specific domain is defined for the websites, the URL
+generated in the email template will use the domain defined on the corresponding website of the
+company.
```
```suggestion
generated in the email template uses the domain defined on the corresponding website of the company.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
-On the other hand, sales orders made by a customer on one of your Odoo e-commerce websites have a
-link established with the website from which the order was made. As a result, the e-mail sent for
-the sales orders uses the domain name defined for the corresponding website to generate the links.
+On the other hand, sales orders made by a customer, on an Odoo eCommerce website, have a link
+established with the website, from which the order was made. As a result, the e-mail sent for sales
+orders uses the domain name defined for the corresponding website to generate the links.
```
```suggestion
On the other hand, sales orders made by a customer, on an Odoo *eCommerce* website, have a link
established with the website, from which the order was made. As a result, the e-mail sent for sales
orders uses the domain name defined for the corresponding website to generate the links.
```

ksc-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,223 @@
+      :alt: Translation of the body of the Appointment Booked template.
+
+.. note::
+   When editing the translations the language set in the database will appear in **bold**.
```
```suggestion
   When editing the translations, the language set in the database appears in **bold**.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
-We will start by addressing issues of outgoing emails (ex: my client has not received my email),
-and then, of incoming emails (ex: I do not receive responses from my customers in the database).
+This document contains an explanation of the most recurring emailing issues in Odoo. First, outgoing
+emails and then, incoming emails. Issues that will be covered regarding outgoing emails include the
+following: when an email isn't sent, common error messages such as hitting the daily email limit,
+:abbr:`SMTP (Simple Mail Transport Protocol)` Errors, unsent emails with no errors and when a
+message is sent late. For incoming messages issues that will be covered, they include when emails
+are not received and what to include when contacting Odoo support.
```
```suggestion
emails. Then, incoming emails. 

Issues that will be covered regarding outgoing emails include the following: when an email isn't
sent, common error messages, such as: hitting the daily email limit, :abbr:`SMTP (Simple Mail
Transport Protocol)` Errors, unsent emails with no errors, and when a message is sent late.

For incoming message issues that will be covered include: when emails are not received, and what to
include when contacting Odoo support.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
 
-What do you have to check if your email is not sent?
-----------------------------------------------------
+What to check if an email is not sent?
+--------------------------------------
```
```suggestion
Email is not sent
-----------------
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+   :alt: Red envelope displayed in chatter.
+
+Unsent emails also appear in the Odoo email queue. In :ref:`Developer mode <developer-mode>` this
+can be accessed by going to :menuselection:`Settings --> Technical Menu --> Emails --> Emails`.
```
clarify which Settings to which you are referring, please

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+
+Unsent emails also appear in the Odoo email queue. In :ref:`Developer mode <developer-mode>` this
+can be accessed by going to :menuselection:`Settings --> Technical Menu --> Emails --> Emails`.
+Unsent emails appear in turquoise, while sent emails will appear in grey.
```
```suggestion
Unsent emails appear in turquoise, while sent emails appear in grey.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
 
-You reached your daily limit:
-*****************************
+Reached the daily limit:
+************************
```
```suggestion
Daily limit reached
*******************
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
-   :align: center
-   :alt: Retry button of an emails
+- Wait until 11 PM (UTC) for the reset, and click on the retry button: The :ref:`Developer mode
+  <developer-mode>` must be activated. Then, go to :menuselection:`Settings --> Technical -->
```
please specify which Settings is being referred to here, as well. thanks!

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+generated by email services are helpful tools to diagnose and troubleshoot email problems.
+
+.. note::
+   The debug menu can be used to investigate sending issues from a database. To access the menu
+   :ref:`Developer mode <developer-mode>` must be activated. Then navigate to the :guilabel:`Debug
+   Menu` in the top right of the menu bar, :menuselection:`Debug menu --> Manage Messages`. The
+   :guilabel:`Manage Messages` menu will open up a list of all the messages sent in a particular
+   record. Within each message there is information on sending, including what type and subtype of
+   the message. Other information includes who the message was sent to and whether Odoo received a
+   bounce-back from an email server.
```
```suggestion
   The debug menu can be used to investigate sending issues from a database. To access the menu,
   :ref:`Developer mode <developer-mode>` must be activated. Once activated, navigate to the
   :guilabel:`Debug Menu` in the top right of the menu bar, :menuselection:`Debug menu -->
   Manage Messages`.
   
   The :guilabel:`Manage Messages` menu opens a list of all the messages sent in a particular
   record. Within each message there is information on sending, including what type, and subtype, of
   the message.
   
   Other information includes to whom the message was sent, and whether Odoo received a bounce-back
   from an email server.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+.. note::
+  In such case, one of the most common reasons is related to :ref:`SPF
+  <email_communication/spf_compliant>` and/or :ref:`DKIM <email_communication/DKIM_compliant>`
+  configuration. Also check to make sure the `mail.bounce.alias` is defined in the *System
```
```suggestion
  configuration. Also, check to make sure the `mail.bounce.alias` is defined in the *System
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+  :menuselection:`Settings App --> Technical Menu --> Parameters --> System Parameters`.
 
-Why is my email sent late?
+Why is an email sent late?
 **************************
```
```suggestion
Email sent late
***************
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+mass mailing, marketing automation, events). The system utility **cron** can be used to schedule
+programs to run automatically at predetermined intervals. Odoo uses that policy in order to avoid
+cluttering the mail servers and, instead, prioritizes the communication. This **cron** is called
+`Mail: Email Queue Manager` and can be accessed in :ref:`Developer mode <developer-mode>` by going
+to :menuselection:`Settings App --> Technical Menu --> Automation --> Scheduled Actions`.
```
```suggestion
`Mail: Email Queue Manager`, and can be accessed in :ref:`Developer mode <developer-mode>` by going
to :menuselection:`Settings App --> Technical Menu --> Automation --> Scheduled Actions`.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
-  might help you on how to get your EML files. Once you get the EML of the email, adding it
-  in the attachment of your ticket is the most efficient way for us to investigate. The support
-  will mainly focus on redundant issues.
+The **EML** (or **headers**) of the email, is the file format containing all the technical
```
```suggestion
The **EML** (or **headers**) of the email is the file format containing all the technical
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
 
-  - Can you provide us with an example of an email that has been correctly forwarded ?
+- Is this simply a reply from an email going out from Odoo?
+- Is there an incoming email server being used, or is email somehow being redirected?
```
```suggestion
- Is there an incoming email server being used, or is the email somehow being redirected?
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+- Is it a generic issue, or is it specific to a use case? If specific to a use case, which one
+  exactly?
+- Is it working as expected? In case the email is sent using Odoo, the bounce email should reach the
+  Odoo database and display the :ref:`red envelope <red_envelop>`.
```
```suggestion
  Odoo database, and display the :ref:`red envelope <red_envelop>`.
```

ksc-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,202 @@
+    .. seealso::
+       The bounce system parameter needs to be set in the technical settings in order for the
+       database to correctly receive bounce messages. To access this setting go to:
+       :menuselection:`Settings --> Technical --> Parameters --> System Parameters`. The parameter
```
please clarify which "Settings" is being referred to here, as well. Thanks! 👍

tiku-odoo replied:
Thanks for your review @ksc-odoo

ksc-odoo reviewed (commented):
@tiku-odoo alrighty - just gave this another review - and I believe, after making these adjustments, you will be ready to move it on to the next stage (Tech Review). Great work 👍 If you have any questions, don't hesitate to reach out!

samueljlieber commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,222 @@
+*transactional emails*, and sometimes contain links redirecting to the Odoo database.
 
 By default, links generated by the database use the dynamic web.base.url key defined in the system
-parameters. More information about this :ref:`parameter <domain-name/web-base-url>`.
+parameters. For more information about this, see :ref:`parameter <domain-name/web-base-url>`.
```
The "web.base.url" formatting here is an interesting one because technically it is a **System Parameter** _record_ with the **Key** field as `web.base.url`and the **Value** field as `https://custom-domain.com`.

(Sorry can't add a suggestion) I think it could be formatted in the following paragraphs first with italics and then no formatting after:

```
...the dynamic *web.base.url* key defined in the system parameters. For more...
```

samueljlieber commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,222 @@
+In order to edit translations, follow these steps from the template:
 
-In order to edit your translations, follow these steps from the template.
+#. To access the translations, first enter :ref:`developer mode <developer-mode>`. To enter
+   developer mode, navigate to :menuselection:`Main Odoo dashboard --> Settings`, scroll down to
+   :guilabel:`Developer Tools` section, and click :guilabel:`Activate the developer mode`.
```
Again I don't feel it is necessary to restate how to enter dev mode after telling the user to enter it with a reference to instructions, but this is just my opinion :)
```suggestion
#. To access the translations, first enter :ref:`developer mode <developer-mode>`.
```

samueljlieber commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,222 @@
+#. To access the translations, first enter :ref:`developer mode <developer-mode>`. To enter
+   developer mode, navigate to :menuselection:`Main Odoo dashboard --> Settings`, scroll down to
+   :guilabel:`Developer Tools` section, and click :guilabel:`Activate the developer mode`.
 
```
No need for empty lines between bullet points of the same level with no other elements between :)
```suggestion
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,208 @@
-- 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription,
+- 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription.
 
-- 50 emails/day for one-app free and trial databases,
+- 50 emails/day for one-app free and trial databases.
 
-- in case of migration, your daily limit might be reset to 50 emails a day.
+- In case of migration, the database daily limit might be reset to 50 emails a day.
```
no need to extra lines between bullet points of the same level

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,208 @@
-time 550: mailbox unavailable).
+When there is an issue with incoming emails, there might not be an indication, per se, in Odoo. It
+is the sending email client, who tries to contact a database, that will get a bounce-back (most of
+the time a `550: mailbox unavailable`).
```
This would be a message displayed on the screen, not a user input, so a :guilabel: is appropriate
```suggestion
the time a :guilabel:`550: mailbox unavailable`).
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,208 @@
+the process on how to get EML/header files. Once the headers of the email are obtained, adding it
+into the Odoo support ticket is the most efficient way for the Odoo Support team to investigate.
 
   .. seealso::
      - `Gmail documentation
        <https://support.google.com/mail/answer/29436>`_
      - `Outlook documentation
        <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
```
this seealso is indented for no reason, scoot it back please 🙂 I don't think there is anything we can do about the URL being over 100 chars, @StraubCreative any ideas?

```suggestion
.. seealso::
   - `Gmail documentation
     <https://support.google.com/mail/answer/29436>`_
   - `Outlook documentation
     <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,208 @@
 
-  - Is it working as expected? In case the email is sent using Odoo, the bounce email should reach
-    the Odoo database and display the :ref:`red envelope <red_envelop>`.
+    .. seealso::
+       The bounce system parameter needs to be set in the technical settings in order for the
+       database to correctly receive bounce messages. To access this setting go to:
+       :menuselection:`Settings app --> Technical --> Parameters --> System Parameters`. The
+       parameter name to select is `mail.bounce.alias`.
```
This seealso is indented too far, please scoot it back and add a :guilabel: to the element with active instruction :)
```suggestion
  .. seealso::
      The bounce system parameter needs to be set in the technical settings in order for the
      database to correctly receive bounce messages. To access this setting go to:
      :menuselection:`Settings app --> Technical --> Parameters --> System Parameters`. The
      parameter name to select is :guilabel:`mail.bounce.alias`.
```

samueljlieber reviewed (changes requested):
Hi @tiku-odoo! This is a very thorough PR nice job with the improvements to `email_template.rst` and `faq.rst`. I have a bunch of technical changes as well as a few points of discussion, looking forward to taking another look after they have been addressed. Let me know if you have any questions, please see my comments below:

Please update the commit message of this PR to follow the convention. You can do this during the _squashing_ step or run `git commit --amend` and update it to: 
`[IMP] Misc: 15.0 email templates / FAQ update`. Once pushed up, you will likely also need to Edit the PR title on GitHub too.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,222 @@
+Settings`, scroll down to :guilabel:`Developer Tools` section, and click :guilabel:`Activate the
+developer mode`.
+
+When clicked, the page refreshes, returns to the main Odoo dashboard, and a "bug" icon is present on
```
Removed this section.

tiku-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,208 @@
-  See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and
-  ``zgrep`` (for the compressed ones) to search through the files.
+.. tip::
+  The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh database
+  contains a list of files containing the logs of the database. The log files are created everyday
+  at 5:00 AM (UTC).
```
Actually turned out to be the bottom two paragraphs out of alignment.

tiku-odoo commented:
@samueljlieber I made the requested changes and corrected a few more capitalizations and removed a list at the end of the Templates.rst page. 

This is ready for a last look! 

Thanks for your thorough reviews. 

:+1:

samueljlieber commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,211 @@
 
    .. image:: email_template/edit-language-template.png
-      :align: left
-      :alt: Edit the language of a template
+      :align: center
+      :alt: Edit the language of a template.
```
Image indented unnecessarily, scoot it back to the beginning of the line please

samueljlieber commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,211 @@
 
    .. image:: email_template/translation-body.png
-      :align: left
-      :alt: Translation of the body of the Application template in the different languages installed.
+      :align: center
+      :alt: Translation of the body of the Appointment Booked template.
```
This one too- scoot it back to the beginning of the line please

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
-   :alt: Red envelope displayed in chatter
+   :alt: Red envelope displayed in chatter.
+
+Unsent emails also appear in the Odoo email queue. In :ref:`Developer mode <developer-mode>` this
```
I don't believe _developer mode_ needs capitalization
```suggestion
Unsent emails also appear in the Odoo email queue. In :ref:`developer mode <developer-mode>` this
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
+  corresponding email documentation </applications/general/email_communication/email_servers>`).
+- Wait until 11 PM (UTC) for the reset, and click on the retry button: The :ref:`developer mode
+  <developer-mode>` must be activated. Then, go to :menuselection:`Settings app --> Technical -->
+  Emails --> Emails`
```
Missing period
```suggestion
  Emails --> Emails`.
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
+This can be mitigated by receiving :ref:`notifications in Odoo
+<discuss_app/notification_preferences>`, instead of emails.
 
 SMTP Error
```
Sentence case title
```suggestion
SMTP error
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
+   Other information includes to whom the message was sent, and whether Odoo received a bounce-back
+   from an email server.
 
 No Error
```
This title too :)
```suggestion
No error
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
+mailing, marketing automation, events). The system utility **cron** can be used to schedule programs
+to run automatically at predetermined intervals. Odoo uses that policy in order to avoid cluttering
+the mail servers and, instead, prioritizes the communication. This **cron** is called
+:guilabel:`Mail: Email Queue Manager`, and can be accessed in :ref:`Developer mode <developer-mode>`
```
```suggestion
:guilabel:`Mail: Email Queue Manager`, and can be accessed in :ref:`developer mode <developer-mode>`
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
 
-If you have this problem on a recurring basis with the same client or the same domain, please
-do not hesitate to contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
+If this a recurring problem with the same client, or the same domain, do not hesitate to contact
```
Missing word
```suggestion
If this is a recurring problem with the same client, or the same domain, do not hesitate to contact
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
-
-  - Is it working as expected? In case the email is sent using Odoo, the bounce email should reach
-    the Odoo database and display the :ref:`red envelope <red_envelop>`.
+  .. seealso::
+      The bounce system parameter needs to be set in the technical settings in order for the
+      database to correctly receive bounce messages. To access this setting go to
+      :menuselection:`Settings app --> Technical --> Parameters --> System Parameters`. The
+      parameter name to select is :guilabel:`mail.bounce.alias`.
```
Indentation and line breaks :)
```suggestion
  .. seealso::
     The bounce system parameter needs to be set in the technical settings in order for the database
     to correctly receive bounce messages. To access this setting go to :menuselection:`Settings app
     --> Technical --> Parameters --> System Parameters`. The parameter name to select is
     :guilabel:`mail.bounce.alias`.
```

samueljlieber reviewed (changes requested):
Hi @tiku-odoo nice job implementing my previous changes, this PR is shaping up nicely! I caught just a couple more corrections, please see below and tag me once more for a final look. Thank you for your continued effort here!

tiku-odoo commented:
@samueljlieber This doc is ready for your re-review. 

I think we've covered all the bases. Let me know if there are any other changes. 

Thanks, 
Tim

samueljlieber commented:
Hi @StraubCreative this PR is ready for your final review 👍 Nice job @tiku-odoo!

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
+  <https://www.odoo.com/help>`_ can still be contacted, if there is a recurring issue with the same
+  client or domain.
 
 Get help from support
```
This section would make more sense as an H2 heading, instead of being under "incoming emails"
```suggestion
Get help from Odoo Support
```

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,174 +1,205 @@
 
 In order to get helped efficiently, please provide as much information as possible. Here is a list
-of what can be helpful:
+of what can be helpful when reaching out to support about an issue:
```
You should link to odoo.com/help (the link to create a support ticket) at the beginning of this section as well
```suggestion
of what can be helpful when reaching out to the Odoo Support team about an issue:
```

meng-odoo reviewed (changes requested):
Hi @tiku-odoo! I gave this doc a final review as someone who has never configured emails/email templates in Odoo before :) I had some questions that I think should be answered/clarified in the doc. I also noticed that for the Email FAQ doc, you often slightly reworded what the previous person had written, when it would be much better to rewrite whole sections in your own words to make sure they make sense.

Please let me know if you have any questions about my comments/suggestions, and then please tag me for another look! Thanks :)

meng-odoo reviewed (changes requested):
Hi @tiku-odoo! I gave this doc a final review as someone who has never configured emails/email templates in Odoo before :) I had some questions that I think should be answered/clarified in the doc. I also noticed that for the Email FAQ doc, you often slightly reworded what the previous person had written, when it would be much better to rewrite whole sections in your own words to make sure they make sense.

Please let me know if you have any questions about my comments/suggestions, and then please tag me for another look! Thanks :)

tiku-odoo reviewed (commented):
@meng-odoo 

THANKS for your review; I know these were two big docs. I've made your requested changes. It's ready for another look from you if you'd like, and then to @samueljlieber for technical RST review. 

I look forward to seeing this published. 

Thanks again for your help. 

Tim

tiku-odoo commented:
Removed references to "What Can I Expect" page per PR# : https://github.com/odoo/documentation/pull/5573

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,225 @@
+
+#. Send a copy of the email headers. The `.EML` file (or **headers**) of the email is the file
+   format containing all the technical information required for an investigation. The documentation
+   from the email provider might explain the process on how to get EML file/header files. Once the
```
```suggestion
   from the email provider might explain how to access the EML file/header files. Once the
```

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,225 @@
+   - Is there an incoming email server being used, or is the email somehow being redirected?
+   - Is there an example of an email that has been correctly forwarded?
+
+#. Provide answers to the following questions:
```
For some reason in the HTML version, it looks like there should be another (blank) paragraph between this line and the previous line

tiku-odoo replied:
Adding this in doesn't make a difference, the lines are skipped.

meng-odoo reviewed (changes requested):
Hi @tiku-odoo! This doc is looking good, much clearer with more thorough explanations--great job! I had some more comments about things to add/clarify this time around, and wording/formatting suggestions. I'd like to review it one more time before sending it to technical review. Thank you!

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,262 @@
+
+   To search the available fields simply click the drop-down next to :guilabel:`Field` and select
+   one or click :guilabel:`Search More`. This will allow the user to select from all of the
+   available fields for the model that the email template is created for.
```
Referring to documentation on Models.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,262 @@
+   :ref:`Views and fields in the developer documentation <developer/fields>`.
+
+.. warning::
+   Custom email templates are out of the scope of Odoo Support.
```
This refers to customizing email templates. I think it's important to make a boundary after discussing how to find the fields within models.

tiku-odoo replied:
I've made a small edit to this to include: "customizing"

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,262 @@
+
+To send mass mails, while in :guilabel:`list` view, select the desired records where the emails are
+to be sent, click the :guilabel:`Action` button, represented by a :guilabel:`⚙️ (gear)`, and select
+the desired email option from the :guilabel:`Action` drop-down menu. If it is possible to send an
```
Noted, thanks.

tiku-odoo replied:
For this I've added in a note that the option can vary based on list view and application.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,262 @@
+.. warning::
+   Custom email templates are out of the scope of Odoo Support.
+
+Default reply on email templates
```
I've made this edit. Thanks.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,262 @@
-the website application, meaning that in this case, if a specific domain is defined for the
-websites, the URL generated in the email template will use the domain defined on the corresponding
-website of the company.
+   For example, a sale order made by a customer, on an Odoo *eCommerce* website, has a link
+   established with the website, from which the order was made. As a result, the e-mail sent for
+   sales orders uses the domain name defined for the corresponding website to generate the links.
```
Looks great thank you!

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,262 @@
+   established with the website, from which the order was made. As a result, the e-mail sent for
+   sales orders uses the domain name defined for the corresponding website to generate the links.
 
 .. caution::
```
Agreed, thank you.

tiku-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,225 @@
 
-- Ask our support team to increase your daily limit. We will analyze the situation of your database
-  depending on (non-exhaustive list):
+  - How many users are in the database?
+  - Which apps are installed?
+  - The bounce rate: the percentage of email addresses that did not receive emails because they were
+    returned by a mail server on its way to the final recipient.
```
Thanks for catching this, much appreciated.

tiku-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,225 @@
-Sales Orders, Invoices, Purchase Orders, etc.) are sent directly.
+Simple Mail Transport Protocol (SMTP) error messages explain why an email wasn't transmitted
+successfully. :abbr:`SMTP (Simple Mail Transport Protocol)` is a protocol to describe the email
+structure, and transmits data from messages over the Internet. The error messages generated by email
+services are helpful tools to diagnose and troubleshoot email problems.
```
These are unique to the service, there are general errors but it is up to each provider to make the error.

meng-odoo replied:
Great, I think it might be helpful to add that as a quick sentence to the end of this paragraph, but it's already kind of implied by the last sentence, so maybe it's not necessary :)

tiku-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,225 @@
+#. Explain the exact flow that is being followed to normally receive those emails in Odoo. Here are
+   examples of questions whose answers can be useful:
+
+   - Is this simply a reply from an email going out from Odoo?
```
I've clarified the questions.

tiku-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,225 @@
 
-  - Is this simply a reply from an email going out from Odoo ?
+   For the following days, they are named with their dates, and then compressed. Use the command
+   :command:`grep` and :command:`zgrep` (for the compressed ones) to search through the files.
```
I've added in clarification.

tiku-odoo commented:
@meng-odoo This doc is ready for another review. I believe it is ready for Zac. 

Thanks for your review. 

Tim :+1:

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
 
-If you have this problem on a recurring basis with the same client or the same domain, please
-do not hesitate to contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
+The :guilabel:`Manage Messages` menu opens a list of all the messages sent in a particular record
```
```suggestion
The :guilabel:`Manage Messages` menu opens a list of all the messages sent in a particular record.
```

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
+delayed task to send emails that are considered "not urgent" (newsletter formats, such as: mass
+mailing, marketing automation, and events). The system utility **cron** can be used to schedule
+programs to run automatically at predetermined intervals. Odoo uses that policy in order to avoid
+cluttering the mail servers and, instead, prioritizes the communication. This **cron** is called
```
```suggestion
cluttering the mail servers and, instead, prioritizes individual communication. This **cron** is called
```

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
+.. important::
+   By default, the *Mass Mailing cron* runs every 60 minutes. This can be changed to no less than 5
+   minutes. However, running the action every 5 minutes would bog down the Odoo database (stress the
+   system), so this is not recommended. To edit the mass mailing cron, select the action
```
```suggestion
   system), so this is not recommended. To edit the mass mailing cron, select the scheduled action
```

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
-     - `Outlook documentation
-       <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
+.. tip::
+   Logs are a stored collection of all the tasks completed in a database. They are a text only
```
```suggestion
   Logs are a stored collection of all the tasks completed in a database. They are a text-only
```

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
-       <https://support.google.com/mail/answer/29436>`_
-     - `Outlook documentation
-       <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
+.. tip::
```
I really think that the first two paragraphs of this Tip should be normal text, not inside the Tip box

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
+   :command:`grep` and :command:`zgrep` (for the compressed ones) to search through the files.
 
-  - Can you provide us with an example of an email that has been correctly forwarded ?
+   .. seealso::
```
This 'See also' box could definitely stand on its own instead of being inside the Tip box

meng-odoo commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,232 @@
+   - Is it working as expected? In case the email is sent using Odoo, the bounce email should reach
+     the Odoo database, and display the :ref:`red envelope <red_envelop>`.
+
+.. seealso::
```
This should be a Note instead of a 'See also'
```suggestion
.. note::
```

meng-odoo reviewed (approved):
Hi @tiku-odoo, great job on this! Thanks so much for all your work in making those changes. I think these docs are very clear and well-organized now. I'm approving with small suggestions. Then please tag Sam for technical review :)

tiku-odoo commented:
@samueljlieber  This doc is ready for your technical review. Thanks, Tim :+1:

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
 
-The first indicator showing you that the email has not been sent is the red envelope next to the
-date and time of the message.
+The first indicator showing that an email has not been sent is the presence of a :guilabel:`red
+envelope`, next to the date and time of the message, located in the chatter.
```
This is optional, but since emojis are used later in the doc I'm suggesting it here. I would also add "_icon_" after the :guilabel: 
```suggestion
The first indicator showing that an email has not been sent is the presence of a red :guilabel:`✉️
(envelope)` icon, next to the date and time of the message, located in the chatter.
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
 .. image:: faq/red-envelop.png
    :align: center
-   :alt: Red envelope displayed in chatter
+   :alt: Red envelope displayed in chatter.
```
Add "icon" descriptor
```suggestion
   :alt: Red envelope icon displayed in chatter.
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
+.. example::
+   This is an example of a 554 SMTP permanent delivery error: `554: delivery error: Sorry, your
+   message to ------@yahoo.com cannot be delivered. This mailbox is disabled (554.30). -
+   mta4471.mail.bf1.yahoo.com
+   --- Below this line is a copy of the message.`
```
This can be on one line, unless you want the "--- Below this line..." to be on a new line, use the `|` line break markup
```suggestion
   mta4471.mail.bf1.yahoo.com --- Below this line is a copy of the message.`
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
-for Odoo to interpret it correctly.
+The debug menu can be used to investigate SMTP sending issues from a database. To access the menu,
+:ref:`developer mode <developer-mode>` must be activated. Once activated, navigate to the
+:menuselection:`Debug Menu` in the top right of the menu bar (the :guilabel:`🪲 (bug)` icon),
```
The 🪲 icon is too recent (2020), please use an [Emoji 1.0](https://emojipedia.org/emoji-1.0) for device compatibility 🙂
```suggestion
:menuselection:`Debug Menu` in the top right of the menu bar (the :guilabel:`🐞 (bug)` icon),
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
-Get help from support
----------------------
+**Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
 
```
Goodbye extra line 👋
```suggestion
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
+   the Odoo Support team to investigate.
+
+   .. seealso::
+      - `Gmail documentation on headers
+        <https://support.google.com/mail/answer/29436>`_
```
Can be one line
```suggestion
      - `Gmail documentation on headers <https://support.google.com/mail/answer/29436>`_
```

samueljlieber commented on content/applications/general/email_communication/faq.rst:
```diff
@@ -1,173 +1,234 @@
+#. Explain the exact flow that is being followed to normally receive those emails in Odoo. Here are
+   examples of questions whose answers can be useful:
+
+   - Is this a notification message from an reply being received in Odoo?
```
I think "an" should be "a"?
```suggestion
   - Is this a notification message from a reply being received in Odoo?
```

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,278 @@
+When a forward slash `/` is typed in the body of an email template, a drop-down menu appears with
+the following options:
+
+**Basic Blocks**
```
I've added in guilabels

tiku-odoo commented:
@samueljlieber Thanks for your initial review. Whenever you can review this again, I think it is ready for zst. 

Thanks, 
Tim :+1:

samueljlieber reviewed (approved):
Looks great, nice work @tiku-odoo! @StraubCreative this PR is ready for merge 🙂 this can be FWP to 16.0 where @tiku-odoo will make additions edits.

StraubCreative reviewed (approved):
Hi @tiku-odoo 
Looks good to me. Couple comments below.
Feel free to merge whenever, thanks!

@robodoo delegate=tiku-odoo

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,276 @@
+When a forward slash `/` is typed in the body of an email template, a drop-down menu appears with
+the following options:
+
+:guilabel:`Basic Blocks`
```
Noted, I will keep that in mind for future revisions.

tiku-odoo replied:
Have this task to complete expanding on this subject: https://www.odoo.com/web#menu_id=4720&cids=3&action=333&active_id=3835&model=project.task&view_type=form&id=3501821

tiku-odoo replied:
This section will be expanded in the doc for Odoo 16.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,276 @@
+- :guilabel:`Appointment`: Add a specific appointment.
+- :guilabel:`Calendar`: Schedule an appointment.
+
+:guilabel:`Navigation`
```
@samueljlieber and I spoke about this and determined there isn't any further nesting we can do here based on the GUI the HTML render looks exactly like the dropdown in production.

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,276 @@
+   :align: center
+   :alt: Powerbox feature in the email template.
+
+XML/HTML code editor
```
Created task to expand on this in future: https://www.odoo.com/web#menu_id=4720&cids=3&action=333&active_id=3835&model=project.task&view_type=form&id=3501821

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,276 @@
-   The email templates use QWeb. The composer allows you to edit emails in their final rendering,
-   making customizations more robust as you don’t have to edit code.
+   Email templates in Odoo use QWeb or XML, which allows the user to edit emails in their final
+   rendering, making customizations more robust, without having to edit any code whatsoever.
```
Added in clarification.

tiku-odoo commented:
@robodoo r+

tiku-odoo commented:
@StraubCreative  All set -- this one is being merged. Thanks for your work on it! Now all the email docs are up-to-date for Odoo 14/Odoo 15 :+1:

tiku-odoo commented on content/applications/general/email_communication/email_template.rst:
```diff
@@ -1,98 +1,276 @@
-   The email templates use QWeb. The composer allows you to edit emails in their final rendering,
-   making customizations more robust as you don’t have to edit code.
+   Email templates in Odoo use QWeb or XML, which allows the user to edit emails in their final
+   rendering, making customizations more robust, without having to edit any code whatsoever.
```
Added in explanation.

StraubCreative commented:
@robodoo r-

StraubCreative commented:
@fw-bot ignore saas-15.4

StraubCreative commented:
@robodoo r+
